### ISSUE TITLE
http 403

### ISSUE THREAD

**Issue by glow995** [Scope-IT/marksman#25]
After trying to run the marksman I get the below error, not sure where the problem is. The API key was created from the SnipeIT user's "ManageAPIKeys"

"C:\Program Files (x86)\Scope-IT\Marksman>Marksman
1/8/2019 12:32:52 PM: Started application.
HTTP 403: Unauthorized. Please check the API key value in your <appSettings>
block of the Marksman config file and ensure it has been set to a valid key.
ERROR: Could not connect to SnipeIT database instance.
Total program execution time 1522ms.
1/8/2019 12:32:53 PM: Exiting application."

**Comment by hdrkhls** [comment]
I get the same problem,

I've generated the API code with the command "php artisan passport:client --personal", and I've set it in the marksman.exe.config config file, and when i started the marksman i got the same error, maybe someone can help me to, thanks

**Comment by danielhogg** [comment]
I am unfamiliar with Laraval's API system using artisan, and I am not sure if it is compatible.

Can you try creating an API key manually in your SnipeIT dashboard? In the settings in the top right, there should be a button called 'Manage API Keys'. Try generating an API key this way, and then put it into your config file.

**Comment by hdrkhls** [comment]
I've tried it, and have set it on the config file but when i started marksman i got the same error
![image](https://user-images.githubusercontent.com/54937107/64393387-552bcf00-d07b-11e9-8b3d-fddcc99a8564.png)
![image](https://user-images.githubusercontent.com/54937107/64393412-6e348000-d07b-11e9-88de-15eb38645b41.png)
that's the API code right ?, i set that on my config file 
![image](https://user-images.githubusercontent.com/54937107/64393439-92905c80-d07b-11e9-831f-725cafdaae9a.png)

and when i run the marksman, i still get the same error

![image](https://user-images.githubusercontent.com/54937107/64393458-afc52b00-d07b-11e9-866e-aed78a88a156.png)

**Comment by Ziggista** [comment]
@hdrkhls I believe that your missing a "/api/v1" from the ned of your BaseURI value field. It should look like
"http://192.100.100.191/api/v1"

**Comment by hdrkhls** [comment]
i don't think so, because i've tried to write /api/v1 in my BaseURI before, but i still get same error

**Comment by hdrkhls** [comment]
@Ziggista thanks, but now i get this massage 
![image](https://user-images.githubusercontent.com/54937107/64407922-bc637680-d0af-11e9-8059-9888e4531135.png)
that's work or not ?

**Comment by Ziggista** [comment]
@hdrkhls Looks like your getting further than you were before. The error points towards the Manufacturer been null.
What is the output of 
get-wmiobject -Query "SELECT * FROM Win32_ComputerSystem" 
in Powershell?

![image](https://user-images.githubusercontent.com/31128905/64409022-ab6c3280-d0ba-11e9-9b78-3dabf0bcb2ae.png)

**Comment by hdrkhls** [comment]
![image](https://user-images.githubusercontent.com/54937107/64409126-76f47880-d0b2-11e9-88a5-67a4dcda6fde.png)
that's my output in powershell

**Comment by Ziggista** [comment]
@hdrkhls see issue #9 This is the same error. What happens when you download and install the latest release?

**Comment by KochsPet** [comment]
Hi there,
I have the same error as described above. I created the API key in the web interface and in the shell. Both fail.
I don't find the mistake. The BaseURI appears to be correct.

![grafik](https://user-images.githubusercontent.com/12435443/72711882-3f223500-3b6a-11ea-8406-29b0ed196400.png)
